APITest/APITest/Managers/ConfigManager.cs

```diff
@@ -9,5 +9,7 @@ public class ConfigManager
         protected IConfigurationRoot Config => new ConfigurationBuilder()
```

Comment:
You can pass all parameters in a single line while overall line length is less than 110 characters